src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,11 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 ++++++
 
+0.5.146
++++++++
+* Vendor new SDK and bump API version to 2023-05-02-preview.
```

Comment:
This '2023-05-02-preview SDK vendor' was in 0.5.143, not part of your PR, could you remove it?